2020Robot/src/main/java/frc/robot/core751/commands/Drivetrain/ArcadeDrive.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.core751.commands.Drivetrain;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.core751.subsystems.DifferentialDriveTrain;
+
+public class ArcadeDrive extends CommandBase {
+
+    private Joystick driveStick;
+    private DifferentialDriveTrain differentialDriveTrain;
+
+    public void ArcadeDrive(int port, DifferentialDriveTrain differentialDriveTrain) {
+        this.driveStick = new Joystick(port);
```

Comment:
The port binding for this might be something we should put in core751 because (at least for west coast drive) the control joystick hasn't changed from year to year, meaning that it might be useful to structure it so that it's in core751. Whether this is via a separate core751 Constants class or something else I don't know. But I think it should be at least considered.

Reply:
I thought about this but based off of wpis comments it seems best practice is to put all the bindings in Constants.java